app/brief_utils.py

```diff
@@ -40,3 +51,57 @@ def get_supplier_service_eligible_for_brief(supplier, brief):
     services = services.filter(Service.supplier_code == supplier.code)
 
     return services.first()
+
+
+def check_seller_emails(brief_data, errs):
+    seller_selector = brief_data.get('sellerSelector', None)
+    if seller_selector == 'allSellers':
+        return None
+
+    seller_email_key = None
+    emails_to_check = []
+    # empty checking
+    if seller_selector == 'oneSeller':
+        seller_email_key = 'sellerEmail'
+        if not brief_data.get(seller_email_key, None):
+            return {seller_email_key: 'answer_required'}
+
+        emails_to_check = [brief_data[seller_email_key]]
+    elif seller_selector == 'someSellers':
+        seller_email_key = 'sellerEmailList'
+        at_least_one_entered = False
+        emails_to_check = brief_data.get(seller_email_key, [])
+        for email in emails_to_check:
```

Comment:
Are you just testing if there's something in the list? How about:
```
if not emails_to_check:
    return {seller_email_key: 'answer_required'}
```

Reply:
Empty list is tested by at_least_one_entered and get return default of []

Reply:
fixed